Metagenomics and Microbial Survey
======
* [Slides](https://github.com/cursobioinfo/BioinformaticsCourse/blob/main/Lectures/Section6)
* [Jupyter Notebook](https://github.com/cursobioinfo/BioinformaticsCourse/blob/main/Lectures/Section6)
* [Lecture Recording](https://github.com/cursobioinfo/BioinformaticsCourse/blob/main/Lectures/Section6)

# Instructor
* Dante Travisany

# Description
The first part of the session will give an overview of germline as well as somatic variant calling. Different types of genomic variation, like single nucleotide variants, short insertions, and deletions as well as larger structural variants will be covered. We will also look at typical scenarios from medical research and diagnostics.

In the second and larger part of this session, we will cover in-depth the analysis steps required to analyze a germline trio dataset, which consists of the exome of a patient along with the exomes of its parents. The students will be enabled to conduct such analyses comprehensively, starting from raw reads and ending with an annotated excel file. The variant calling workflow will combine and revisit many aspects of the previous sessions.

# Objectives
1.- Understand the basics of mapping and alignment.

2.- An introduction to genetic variants and population genomics.  

3.- Apply the lessons to a real-life example. 

# Critical Concepts
Burrows-Wheeler transform, CIGAR, SAM/BAM, exome.  
