Introduction to Genomics and NGS
======
* [Slides](https://github.com/cursobioinfo/BioinformaticsCourse/blob/main/Lectures/Section5)
* [Jupyter Notebook](https://github.com/cursobioinfo/BioinformaticsCourse/blob/main/Lectures/Section5)
* [Lecture Recording](https://github.com/cursobioinfo/BioinformaticsCourse/blob/main/Lectures/Section5)

# Instructor
* Dante Travisany

# Description
* Theory class: Introduction to bioinformatics with a focus in genomics. Genome complexity covering genome repetitions, heterozygosity, and ploidy. Explanation of the most important Next Generation Sequencing (NGS) from the second and third generations. We will cover throughput, quality, benefits and its pitfalls. For Illumina, library construction and the idea to overpass genome repetitions using different insert sizes. Phred Quality Values and the FASTQ format. 
* Practice session: Will cover a brief introduction to shell terminal, remote connection to a server using SSH, and basic shell commands. We will perform base calling using reads from Oxford Nanopore and analysis of base quality using open-source tools.

# Objectives
1.- Provide to the student the background of genome complexity.

2.- Understand the NGS basis benefits and limitations.

3.- Introduce the student to remote server and computation analysis using the shell.

4.- Provide the tools and ability to understand how good the sequencing run was.

# Critical Concepts
Genome, NGS, base-calling, base quality, Phred-quality values
